metricbeat/module/elasticsearch/pending_tasks/_meta/test/empty.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Love the extensive testing set you added. I assume these files all come ES 6.x? One thing I did with other datasets is to add the version to the file name for later to indicated which version it came from. This also makes it easy to test across major versions.